src/common/snippets/src/op/subgraph.cpp

```diff
@@ -290,33 +300,34 @@ snippets::Schedule snippets::op::Subgraph::generate(ngraph::pass::Manager& opt,
     return {exec_domain, false /*canBeLinearized*/, ptr};
 }
 
-void snippets::op::Subgraph::print() const {
+#ifdef DEBUG_CAPS
+#ifdef ENABLE_OPENVINO_DEBUG
+void snippets::op::Subgraph::debug_verbose(const std::string& annotation) const {
     INTERNAL_OP_SCOPE(Subgraph);
-    remark(13) << "subgraph " << this->get_friendly_name() << " "
-        << this->get_type_name()
-        << " which contains " << this->get_body()->get_ops().size() << " nodes" << std::endl;
+    SNIPPETS_DEBUG << (annotation.empty() ? "" : annotation + ": ")
+                   << this->get_type_name() << " " << this->get_friendly_name()
+                   << " contains " << this->get_body()->get_ops().size() << " nodes";
 
     int qqq = 0;
```

Comment:
Since you are modifying this code anyway, could you please rename "qqq" to something more meaningful "op_order" for example? Also it seems that you can simply use `m_body` instead of `this->get_body()`, `get_friendly_name()` instead of `this->get_friendly_name()` etc.